### PR TITLE
Add the foreignObject element

### PR DESCRIPTION
**Pull request by octronics**
[`foreignObject' documentation](https://www.w3.org/TR/SVG/embedded.html#ForeignObjectElement)

### CODE REVIEW

File: src/node/element/mod.rs [hunk 1]
```diff
@@ -186,6 +186,9 @@ implement! {
 
     #[doc = "A [`use`](https://www.w3.org/TR/SVG/struct.html#UseElement) element."]
     struct Use
+
+    #[doc = "A [`foreignObject'](https://www.w3.org/TR/SVG/embedded.html#ForeignObjectElement) element."]
+    struct ForeignObject
```
Comment (IvanUkhov) [inline src/node/element/mod.rs]: Please keep the list sorted alphabetically.

File: src/node/element/tag.rs [hunk 2]
```diff
@@ -157,6 +157,7 @@ implement! {
     TextPath: "textPath",
     Title: "title",
     Use: "use",
+    ForeignObject: "foreignObject",
```
Comment (IvanUkhov) [inline src/node/element/tag.rs]: The same here.

**Review by IvanUkhov — CHANGES_REQUESTED** [review]
Thank you for the pull request! Please reorder the elements, and I’ll merge it.

File: src/node/element/mod.rs (hunk 1 above)
Comment (octronics) [reply src/node/element/mod.rs]: Done

**Comment by IvanUkhov** [comment]
Thank you!

**Pull request merged**